provider/gcp/gcp_instance.go

```diff
@@ -55,17 +55,30 @@ func (p *GCloud) CreateInstance(ctx *lepton.Context) error {
 	instanceName := c.RunConfig.InstanceName
 	canIPForward := c.RunConfig.CanIPForward
 
+	cimg := &compute.AttachedDiskInitializeParams{
+		SourceImage: imageName,
+	}
+
+	if c.BaseVolumeSz != "" {
+		size, err := lepton.GetSizeInGb(c.BaseVolumeSz)
+		if err != nil {
+			return fmt.Errorf("cannot get volume size: %v", err)
+		}
+		c.BaseVolumeSz = "" // create minimum-sized local volume
```

Comment:
This is not needed (it's only needed when the command being executed creates a local image/volume file, which is not the case for `ops instance create`).